--- ctypeslib/ctypeslib/codegen/codegenerator.py.orig	2015-10-06 18:47:43.000000000 +0200
+++ ctypeslib/ctypeslib/codegen/codegenerator.py	2015-10-23 23:22:54.836001586 +0200
@@ -65,7 +65,7 @@
 
 def storage(t):
     # return the size and alignment of a type
-    if isinstance(t, typedesc.Typedef):
+    if isinstance(t, typedesc.Typedef) or isinstance(t, typedesc.CvQualifiedType):
         return storage(t.typ)
     elif isinstance(t, typedesc.ArrayType):
         s, a = storage(t.typ)
